packages/distribution/addon/components/cd-inquiry-new-form/select.hbs

```diff
@@ -1,14 +1,22 @@
 {{#unless this.showAllServices}}
   <div class="uk-margin-bottom uk-button-group" data-test-group-toggle-bar>
     {{#each-in this.config.new.types as |slug config|}}
-      {{#unless config.disabled}}
+      {{#if config.notAvailableText}}
```

Comment:
The config is implicitly `disabled` by setting `notAbailableText` because you dont check it.
Would make sense to check `disabled` so buttons can be enabled without having to remove the text.